tests/LazyPropertyTestAsset/BClass.php

```diff
@@ -1,43 +1,30 @@
 <?php
-/*
- * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
- * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
- * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
- * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
- * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
- * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
- * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
- * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
- * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
- * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
- * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- *
- * This software consists of voluntary contributions made by many individuals
- * and is licensed under the MIT license.
- */
+
+declare(strict_types=1);
 
 namespace LazyPropertyTestAsset;
 
 use LazyProperty\LazyPropertiesTrait;
 
 /**
  * Mixed properties
- *
- * @author Marco Pivetta <ocramius@gmail.com>
  */
 class BClass extends AClass
 {
     use LazyPropertiesTrait;
 
+    /** @var string */
     private $private;
 
-    public function initBLazyProperties()
+    public function initBLazyProperties() : void
     {
         $this->initLazyProperties(['private']);
     }
 
-    private function getPrivate()
+    // phpcs:disable SlevomatCodingStandard.Classes.UnusedPrivateElements
```

Comment:
Better idea: get rid of `SlevomatCodingStandard.Classes.UnusedPrivateElements` by excluding this rule overall